### PR TITLE
Fix cohort link to patient page when length exceeds browser limit

### PR DESCRIPTION
**Pull request by alisman**
Fixes https://github.com/cbioportal/cbioportal/issues/8317

Chrome changed their url limit.  Our threshhold for appending list of sample ids was too high
image

### CODE REVIEW

File: src/pages/studyView/studyPageHeader/ActionButtons.tsx [hunk 1]
```diff
@@ -68,8 +68,15 @@ export default class ActionButtons extends React.Component<
...
-            if (navCaseIds.length >= MAXIMUM_NAV_CASE_IDS_IN_URL) {
+            const url = getPatientViewUrl(
+                firstPatient.studyId,
+                firstPatient.patientId,
+                navCaseIds
+            );
+
+            // if length is great than 4500 it will crash browsers
+            // so use alternative method passing ids
+            if (url.length > 4500) {
```
Comment (onursumer) [inline src/pages/studyView/studyPageHeader/ActionButtons.tsx:79]: should we also define a constant for this? something like:
```JavaScript
const MAX_ALLOWED_URL_LENGTH = 4500;
```

File: src/pages/studyView/studyPageHeader/ActionButtons.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ export interface ActionButtonsProps {
     appStore: AppStore;
 }
 
-export const MAXIMUM_NAV_CASE_IDS_IN_URL = 2000;
+export const MAXIMUM_NAV_CASE_IDS_IN_URL = 100;
```
Comment (inodb) [inline src/pages/studyView/studyPageHeader/ActionButtons.tsx:23]: can prolly remove this variable now that we use `4500` characters limit directly for the url length check

File: src/pages/studyView/studyPageHeader/ActionButtons.tsx [hunk 3]
```diff
@@ -82,11 +89,7 @@ export default class ActionButtons extends React.Component<
             } else {
                 // add navCaseIds into url if number of cases less than MAXIMUM_NAV_CASE_IDS_IN_URL
```
Comment (inodb) [inline src/pages/studyView/studyPageHeader/ActionButtons.tsx:90]:
```suggestion
```

File: src/pages/studyView/studyPageHeader/ActionButtons.tsx (hunk 1 above)
Comment (alisman) [reply src/pages/studyView/studyPageHeader/ActionButtons.tsx]: done

**Pull request merged**